subt/docker/robotika/Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM osrf/subt-virtual-testbed:subt_solution_latest
+FROM robotika/subt-base:2020-04-21
```

Comment:
I would avoid this hard-coded dependency and use `latest` instead, but you can convince me. Also `2020-04-21` is not really version for me (I know that for some competitions it is the case, but maybe there is only one docker file generator from master only so it is not problem) ... BTW I overlooked this until it failed to build

Reply:
This is very intentional. Using `latest` you never know what you are building it against. The difference is similar to running `pip install opencv-python` vs `pip install opencv-python==4.2.1`. The goal here is that each member of our team runs the same thing as much as possible.